CMakeLists.txt

```diff
@@ -4,11 +4,11 @@
 #
 cmake_minimum_required(VERSION 3.17)
 
-set(FAMILY "rp2040")
```

Comment:
(commit message) the title says which problem you had, not what this commit does

Reply:
Ah, I get it now, so you mean the commit title should reflect what happened. So this should be fine then, right?

> Fix cmake initialization of sub-projects
>
> Pico SDK is initialized by TinyUSB setup already. TinyUSB only needs
> to know where it can find PicoSDK.

Reply:
Pushed that